js/data.js

```diff
@@ -6,3 +6,45 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var formInput = document.querySelector('.input-form');
+
+formInput.addEventListener('submit', submitUserInput);
+
+function submitUserInput(event) {
+  event.preventDefault();
+  var image = document.querySelector('img');
+  var inputForm = document.querySelector('.input-form');
+  var input = inputForm.elements;
+  var dataObject = {};
+  for (var i = 0; i < input.length; i++) {
+    if (input[i].name === 'title') {
+      dataObject.title = input[i].value;
+    } else if (input[i].name === 'photo-url') {
+      dataObject.url = input[i].value;
+    } else if (input[i].name === 'notes') {
+      dataObject.notes = input[i].value;
+    }
+  }
+  data.nextEntryId += 1;
```

Comment:
The submit event handler should be in `main.js`, once you make your change I requested, there will no longer be any `localStorage` work in it